File: app/src/main/java/com/ssynhtn/waveview/MainActivity.java
package com.ssynhtn.waveview;

import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    private static final String TAG = MainActivity.class.getSimpleName();

    private WaveView waveView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        waveView = findViewById(R.id.wave_three);
        waveView.addDefaultWaves(2, 1);

        waveView.startAnimation();
    }

    @Override
    protected void onResume() {
        super.onResume();

        waveView.resumeAnimation();
    }

    @Override
    protected void onPause() {
        super.onPause();

        waveView.pauseAnimation();
    }
}

File: app/src/main/java/com/ssynhtn/waveview/MainActivity.kt
package com.ssynhtn.waveview

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    private lateinit var waveView: WaveView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        waveView = findViewById(R.id.wave_three)
        waveView.addDefaultWaves(2, 1)

        waveView.startAnimation()
    }

    override fun onResume() {
        super.onResume()

        waveView.resumeAnimation()
    }

    override fun onPause() {
        super.onPause()

        waveView.pauseAnimation()
    }
}
